hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/ContractAutoExpirySpecs.java

```diff
@@ -92,7 +92,7 @@ public static void main(String... args) {
 
     @Override
     public List<HapiSpec> getSpecsInSuite() {
-        return List.of(
+        return List.of(new HapiSpec[] {
```

Comment:
Matt, why did you add this array creation back in? I had taken it out because SonarLint was complaining about it. List.of(_) can take the elements directly as arguments, instead of passing an array.

Reply:
Sorry. It's easier to comment out tests individually with an array because the last array element can be followed by a comma, which takes about a third of the three seconds it takes to move a parenthesis